bundles/ui/org.eclipse.smarthome.ui.login/.settings/org.maven.ide.eclipse.prefs

```diff
@@ -0,0 +1,9 @@
+#Fri Feb 19 20:27:57 CET 2010
+activeProfiles=
+eclipse.preferences.version=1
+fullBuildGoals=process-test-resources
+includeModules=false
+resolveWorkspaceProjects=true
+resourceFilterGoals=process-resources resources\:testResources
+skipCompilerPlugin=true
+version=1
```

Comment:
Please remove this file because it causes the default Eclipse project settings to be overridden. See also #6106.